src/introduction/run-release.md

```diff
@@ -0,0 +1,60 @@
+# Running Exercises with the CHERI Software Release
+
+The [University of Cambridge CHERI Software Release](./get.md#ucam_dist)
+contains pre-built versions of all of the exercises described in this course,
+allowing you to work throught the exercises without first needing to build them
+from sources.
+
+With the CHERI Software Release installed, run either the CHERI-RISCV
+or Morello simulators using one of the following commands:
```

Comment:
Throughout I’m left slightly cautious about the use of terms like “simulator”, “emulator”, and “model”. It might be useful to use fewer of those words .. or define them more clearly?